msg/MotionPlanRequest.msg

```diff
@@ -51,8 +51,8 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
-# Maximum cartesian speed for the given end effector.
+# Maximum cartesian speed for the given link.
 # If max_cartesian_speed <= 0 the trajectory is not modified.
 # These fields require the following planning request adapter: default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed
```

Comment:
``default_planner_request_adapters/setMaxCartesianLinkSpeed``
btw I think it should be changed to **S**etMaxCartesianLinkSpeed (capital S) in https://github.com/ros-planning/moveit/pull/2674 for consistency with the names of other planner_request_adapters

Reply:
@Thieso We forgot one last detail in the comments:
default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed -> default_planner_request_adapters/LimitMaxCartesianLinkSpeed

Reply:
Good catch, thanks.